common/tsconfig.json

```diff
@@ -1,25 +1,5 @@
 {
-  "compilerOptions": {
-    "composite": true,
-    "outDir": "lib",
-    "rootDir": ".",
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "baseUrl": ".",
-    "paths": {},
-    "allowJs": false,
-    "skipLibCheck": true,
-    "strict": false,
-    "forceConsistentCasingInFileNames": true,
-    "esModuleInterop": true,
-    "module": "commonjs",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "jsx": "preserve",
-    "strictNullChecks": true,
-    "downlevelIteration": true
-  },
-  "exclude": ["node_modules", "lib"],
+  "extends": "../config/tsconfig-base",
+  "exclude": ["node_modules"],
```

Comment:
Figure out how to add this to the shared config?

Reply:
would `**/node_modules` do it?

Reply:
I also think I previously had issues with getting _out_ of the `config` folder, this wildcard works it out! 🙏

Reply:
Annnnd now I'm confused again, it just seems to ignore it by default. Let's discuss tomorrow! When I try to add

```
  "exclude": ["**/node_modules"],
  "include": ["**/*.ts", "**/*.tsx"],
```

I get 

```
 No inputs were found in config file '/Users/cantinr/Projects/wellcomecollection.org/cardigan/tsconfig.json'. Specified 'include' paths were '["../config/**/next-env.d.ts","../config/**/**/*.ts","../config/**/**/*.tsx"]' and 'exclude' paths were '["../config/**/node_modules"]'.
```

which would intimate that it never finds the other node_modules. I fully removed it and nothing was flagged. It being end of day I'll leave it be for now but let's discuss..

Reply:
We added a `/` prefix and that figured it out. From the [extends](https://www.typescriptlang.org/tsconfig/#extends) documentation:

> Properties with relative paths found in the configuration file, which aren’t excluded from inheritance, will be resolved relative to the configuration file they originated in.